mhr-api/src/mhr_api/utils/registration_validator.py

```diff
@@ -276,16 +276,17 @@ def validate_permit(
             error_msg += validator_utils.validate_registration_state(
                 registration, staff, MhrRegistrationTypes.PERMIT, MhrDocumentTypes.REG_103, json_data
             )
-            if registration and group_name and group_name == MANUFACTURER_GROUP:
```

Comment:
From the ticket description the removal of the restriction applies only to dealers. MANUFACTURER_GROUP applies to QS manufactures, which I do not think has changed.

Reply:
Definitely correct, only Dealers are changing.

Reply:
Yep! I will update it.